tcf_website/static/base/tcf-style.css

```diff
@@ -19,10 +19,15 @@ h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
   /* Default heading font: Lato (others are fallbacks) */
   font-family: 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  font-weight: 700;
```

Comment:
Are we making the font-weight thinner?

Reply:
yeah I talked to you abt that and you said it looked better thinner
I agree
it is global but I think it looks good where I checked

Reply:
gotcha yeah I just forgot which branch you did it on